packages/graphql-transformers-e2e-tests/resources/jsonServer/yarn.lock

```diff
@@ -215,7 +215,7 @@
     "@aws-cdk/cx-api" "1.124.0"
     constructs "^3.3.69"
 
-"@aws-cdk/aws-lambda@1.124.0", "@aws-cdk/aws-lambda@~1.124.0":
+"@aws-cdk/aws-lambda@1.124.0", "@aws-cdk/aws-lambda@~1.130.0":
```

Comment:
There is a reference to 1.24.0 still, any idea where it might be coming from?

Reply:
Let me regenerate that yarn.lock file as well